protocol/error.go

```diff
@@ -26,8 +34,11 @@ const (
 	ErrorCouldNotVerify
 )
 
-// ErrorResponses contains error codes that
-// a response can omit the DirectoryResponse.
+// ErrorResponses contains the server responses indicating that
+// a client request could not be processed due to a
+// malformed client request or an internal server error.
+// Server response messages containing one of these errors therefore
+// omit the DirectoryResponse.
```

Comment:
> ErrorCouldNotVerify is another good candidate

It would be removed in https://github.com/coniks-sys/coniks-go/pull/113/files#diff-78429462c41fd4a6aab3cc13baa05919L28. 

> Maybe it makes sense to treat the non-errors more as different results, e.g. change the prefix to something like "Req-" for all request results (including Success?) and "Check-" for all consistency check results? So we'd be dealing with error codes such as "ReqNameNotFound" or "CheckBadCommitment". 

Yes, it makes sense to me, maybe we can keep `Error-` prefix for error codes in `ErrorResponses`.